src/Main.elm

```diff
@@ -720,7 +720,7 @@ onDrop msg =
 
 onDragOver : msg -> Attribute msg
 onDragOver msg =
-    preventDefaultOn "dragover" <| Json.Decode.map (\x -> ( x, True )) <| Json.Decode.succeed msg
+    preventDefaultOn "dragover" <| Json.Decode.succeed ((\x -> ( x, True )) <| msg)
```

Comment:
https://package.elm-lang.org/packages/elm/json/latest/Json-Decode#succeed

😮 

Looks working the D&D on `npm run dev`